src/pretix/static/pretixcontrol/js/ui/main.js

```diff
@@ -301,26 +301,36 @@ var form_handlers = function (el) {
         var rgb = $(this).colorpicker('color').toRGB();
```

Comment:
To fix the described bug regarding destroy, we also need to ignore doubled changeColor-events.

```suggestion
        if (e.type == 'changeColor' && !e.value) {
            return;
        }
        var rgb = $(this).colorpicker('color').toRGB();
```